### ISSUE TITLE
Update the bears to match with the examples in the native bears tutorial

### ISSUE THREAD

**Issue by Kagamihime** [coala-analyzer/coala-tutorial#10]
The code in `HelloWorldBear.py` and `CommunicationBear.py` is out of date compared to the example codes presented at http://api.coala.io/en/latest/Developers/Writing_Native_Bears.html.

**Comment by Kagamihime** [comment]
I'd like to be assigned myself to this issue (unless it will be marked as a newcomer issue of course)

**Comment by Kagamihime** [comment]
@Makman2 I've got an issue: in `HelloWorldBear.py`, if I keep (as in the tutorial for now) `logging.debug("Hello World! Checking file", filename, ".")`, I get an error but not if I use the `format` method (as I intended to do in my previous PR, https://github.com/coala/coala/pull/5100).
What should I use for now? I guess it will be solved soon via https://github.com/coala/coala/issues/5109 but I'm not sure about what to do for now.
